src/client/pages/PasscodeEmailSent.tsx

```diff
@@ -0,0 +1,170 @@
+import React, {
+	PropsWithChildren,
+	ReactNode,
+	useState,
+	useEffect,
+} from 'react';
+import { MainBodyText } from '@/client/components/MainBodyText';
+import { MainForm } from '@/client/components/MainForm';
+import { EmailInput } from '@/client/components/EmailInput';
+import { ExternalLink } from '@/client/components/ExternalLink';
+import { buildUrl } from '@/shared/lib/routeUtils';
+import locations from '@/shared/lib/locations';
+import { SUPPORT_EMAIL } from '@/shared/model/Configuration';
+import {
+	InformationBox,
+	InformationBoxText,
+} from '@/client/components/InformationBox';
+import { css } from '@emotion/react';
+import { FieldError } from '@/shared/model/ClientState';
+import { logger } from '@/client/lib/clientSideLogger';
+import { MinimalLayout } from '@/client/layouts/MinimalLayout';
+import ThemedLink from '@/client/components/ThemedLink';
+import { PasscodeInput } from '@/client/components/PasscodeInput';
+
+type Props = {
+	passcodeAction: string;
+	changeEmailPage?: string;
+	email?: string;
+	errorMessage?: string;
+	fieldErrors?: FieldError[];
+	formError?: string;
+	formTrackingName?: string;
+	passcode?: string;
+	queryString?: string;
+	recaptchaSiteKey?: string;
+	showSuccess?: boolean;
+	timeUntilTokenExpiry?: number;
+};
+
+const sendAgainFormWrapperStyles = css`
+	white-space: nowrap;
+`;
+
+export const PasscodeEmailSent = ({
+	changeEmailPage,
+	children,
+	email,
+	errorMessage,
+	fieldErrors,
+	formError,
+	formTrackingName,
+	passcode,
+	passcodeAction,
+	queryString,
+	recaptchaSiteKey,
+	showSuccess,
+	timeUntilTokenExpiry,
+}: PropsWithChildren<Props>) => {
+	const [recaptchaErrorMessage, setRecaptchaErrorMessage] = useState('');
+	const [recaptchaErrorContext, setRecaptchaErrorContext] =
+		useState<ReactNode>(null);
+
+	useEffect(() => {
+		// we only want this to run in the browser as window is not
+		// defined on the server
+		// and we also check that the expiry time exists so that
+		// we redirect to the session expired page
+		// if the token expires while the user is on the current page
+		if (typeof window !== 'undefined' && timeUntilTokenExpiry) {
+			logger.info(`Welcome page: loaded successfully`, undefined, {
+				timeUntilTokenExpiry,
+			});
+			setTimeout(() => {
+				logger.info(
+					`Welcome page: redirecting to token expired page`,
+					undefined,
+					{ timeUntilTokenExpiry },
+				);
+				window.location.replace(buildUrl('/welcome/expired'));
+			}, timeUntilTokenExpiry);
+		}
+	}, [timeUntilTokenExpiry]);
+
+	// autofocus the code input field when the page loads
+	useEffect(() => {
+		if (typeof window !== 'undefined') {
+			const codeInput: HTMLInputElement | null =
+				window.document.querySelector('input[name="code"]');
+
+			if (codeInput) {
+				codeInput.focus();
+			}
+		}
+	}, []);
+
+	return (
+		<MinimalLayout
+			pageHeader="Enter your code"
+			successOverride={showSuccess ? 'Email sent' : undefined}
```

Comment:
I think we should say 'Verification code sent' here... makes it obvious what the action was.

Reply:
Thats what it used to say, or something along those lines, it was simplified.

Reply:
Updated to `Email with verification code sent`